api/meetings/meetingsRouter.js

```diff
@@ -169,7 +182,7 @@ function validProfileID(req, res, next) {
         req.profile = profile;
         next();
       } else {
-        res.status(404).json({
+        res.status(400).json({
```

Comment:
400 status code makes a lot more sense in that situation

Reply:
I just pushed a new commit fixing this. Thank you Mina!